v-next/hardhat/src/types/solidity/build-system.ts

```diff
@@ -45,6 +45,11 @@ export interface BuildOptions {
    * If `true`, the build process doesn't print any output.
    */
   quiet?: boolean;
+
+  /**
+   * If `true`, the root files will be instrumented before compilation.
```

Comment:
I think this comment is out of sync with respect to the implementation. In particular, which files get instrumented.

Reply:
```suggestion
   * If `true`, the project files will be instrumented before compilation.
```